packages/lib/services/interop/types.ts

```diff
@@ -99,6 +105,12 @@ export interface ExportOptions {
 	plugins?: PluginStates;
 	customCss?: string;
 	packIntoSingleFile?: boolean;
+
+	// Overrides the settings above and directly supplies
+	// an exporter class. This may be used in environments
+	// where the default look-up logic fails (e.g. no dynamic
+	// importing).
+	exporter?: InteropService_Exporter_Base;
```

Comment:
We indeed want to move away from dynamic imports. How difficult would it be to refactor InteropService so that requireDynamic is not needed? I guess you could pre-import all the modules and create a map from it like it's been done for the plugins in MdToHtml.ts?

Reply:
I've opened a pull request that does something similar: https://github.com/laurent22/joplin/pull/8454/files

Because we're already using a list of metadata objects, the pull request attaches the pre-imported modules to the objects in the list. (Rather than creating a map).